src/auth/googleauth.ts

```diff
@@ -634,4 +643,59 @@ export class GoogleAuth {
           if (callback) callback(null, body);
         });
   }
+
+
+  /**
+   * Returns the contents of the metadata of GC instance
```

Comment:
nit: the doc needs to be expanded a bit as this is public API. It seems the behavior is different depending upon whether we have user provided `jsonContent` or whether we are fetching from the metadata service. There needs to be a more detailed explanation of what this function does.

Reply:
Done.